dist/css/bootstrap.css

```diff
@@ -5264,7 +5264,7 @@ button.close {
   color: #fff;
   text-align: center;
   text-decoration: none;
-  background-color: rgba(0, 0, 0, .9);
+  background-color: #000;
```

Comment:
Sure you want to include this change? I don't know what the difference between `#000` and `rgba(0,0,0,.9)` is (mostly I don't know what the `.9` does)